cmd/client-s3.go

```diff
@@ -223,11 +223,12 @@ func newFactory() func(config *Config) (Client, *probe.Error) {
 			var e error
 
 			options := minio.Options{
-				Creds:        creds,
-				Secure:       useTLS,
-				Region:       os.Getenv("MC_REGION"),
-				BucketLookup: config.Lookup,
-				Transport:    transport,
+				Creds:           creds,
+				Secure:          useTLS,
+				Region:          os.Getenv("MC_REGION"),
+				BucketLookup:    config.Lookup,
+				Transport:       transport,
+				TrailingHeaders: config.TrailingHeaders && isSigV4,
```

Comment:
Since trailing is false by default, can we return an error if we have trailing enabled and not sigv4 ?

Reply:
Sure

Reply:
I added the check to `mc alias set`. It is not important enough that we should crash it when loading from disk.